tikvrpc/tikvrpc_test.go

```diff
@@ -47,3 +53,87 @@ func TestBatchResponse(t *testing.T) {
 	assert.Nil(t, batchResp)
 	assert.NotNil(t, err)
 }
+
+// https://github.com/pingcap/tidb/issues/51921
+func TestTiDB51921(t *testing.T) {
```

Comment:
Is it necessary to add some loop times to increase the possbility of race if it exists.

Reply:
It's already try each request type repeatedly for 250ms.